src/main/webapp/auth.js

```diff
@@ -58,12 +58,12 @@ function onSignIn(googleUser) {
   // Set cookie that contains idToken to authenticate the user
   // Will automatically delete when the access token expires
   // or when the browser is closed
-  document.cookie = `idToken=${idToken}; expires=${expiryUtcTime}`;
+  addCookie('idToken', idToken, expiryUtcTime);
```

Comment:
This looks clean after wrapping it in a function!

Reply:
Thanks!